fe/fe-core/src/test/java/com/starrocks/journal/bdbje/BDBJEJournalTest.java

```diff
@@ -756,4 +757,22 @@ public void write(DataOutput out) throws IOException {
         Assert.assertEquals(4, journal.getMaxJournalId());
         Assert.assertEquals(1, journal.getMinJournalId());
      }
+
+    @Test
+    public void testJournalWithPrefix(@Mocked BDBEnvironment environment) throws Exception {
```

Comment:
Better use real environment instead of mocking here.

Reply:
done